# yaw, rool, pitch's unit is in 0.1 degree
# duration's unit in 0.1s 

int16 yaw
int16 roll
int16 pitch
int16 duration
# absolute = 1, incremental = 0
bool absolute_or_incremental
bool yaw_cmd_ignore
bool roll_cmd_ignore
bool pitch_cmd_ignore
---
bool result 
